vita3k/modules/SceCommonDialog/SceCommonDialog.cpp

```diff
@@ -1009,7 +1009,7 @@ EXPORT(int, sceSaveDataDialogGetResult, Ptr<SceSaveDataDialogResult> result) {
         return RET_ERROR(SCE_COMMON_DIALOG_ERROR_NULL);
     }
 
-    if (host.common_dialog.status != SCE_COMMON_DIALOG_STATUS_FINISHED && host.common_dialog.substatus != SCE_COMMON_DIALOG_STATUS_FINISHED) {
+    if (host.common_dialog.status != SCE_COMMON_DIALOG_STATUS_FINISHED || host.common_dialog.substatus != SCE_COMMON_DIALOG_STATUS_FINISHED) {
```

Comment:
```suggestion
    if ((host.common_dialog.status != SCE_COMMON_DIALOG_STATUS_FINISHED) || (host.common_dialog.substatus != SCE_COMMON_DIALOG_STATUS_FINISHED)) {
```
maybe ?

Reply:
Nope. https://en.cppreference.com/w/cpp/language/operator_precedence

Reply:
All rpcs3 dev have said me need zdd () if you usine two things like this.
Like if (i == 2 || i == 3) request add () for both for correctly separe it

Reply:
It's probably coding convention in rpcs3 to avoid mistake when adding more expressions.

Reply:
Notre really i have see is normal for all dev, vita3k dev using also this

Reply:
All dev? https://github.com/google/googletest/blob/master/googletest/src/gtest-port.cc#L322 https://github.com/tensorflow/tensorflow/blob/master/tensorflow/cc/framework/cc_op_gen.cc#L60 https://github.com/tensorflow/tensorflow/blob/master/tensorflow/cc/framework/cc_op_gen.cc#L282 https://github.com/google/googletest/blob/master/googletest/src/gtest-port.cc#L1011 https://github.com/Vita3K/Vita3K/blob/master/vita3k/modules/SceGxm/SceGxm.cpp#L1665

Reply:
no, i want said, i seeé majory case explain need ad () for all if (())
vita3k dev have already request me i add this () because have two == inside if

Reply:
I just don't see the need here. If you can give me more proper reason to me, I will add them. Why do you keep bothering with weird idea again and again. I'm tired of this.

Reply:
sorry, i have not wanted tired you :/ i have just explazin exacltyy like other dev have explain me that all :(